examples/auth-with-stytch/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "stytch-nextjs-integration",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start"
+  },
+  "dependencies": {
+    "@stytch/stytch-react": "^3.0.2",
```

Comment:
Could we update all deps?

Reply:
1e605fb

Updated Stytch and other dependencies to their latest versions.